mne/export/_edf.py

```diff
@@ -264,9 +264,7 @@ def _export_raw(fname, raw, physical_range, add_ch_type):
         annots = raw.annotations
         if annots is not None:
             n_annotations = len(raw.annotations)
-            n_annot_chans = int(n_annotations / n_blocks)
-            if np.mod(n_annotations, n_blocks):
-                n_annot_chans += 1
+            n_annot_chans = int(n_annotations / n_blocks) + 1
```

Comment:
I noticed that annotations weren't correctly written to the file (they were basically cropped in some cases) with the current `n_annot_chans` computation.